pkg/kubelet/server/server.go

```diff
@@ -340,8 +345,8 @@ func (s *Server) InstallAuthFilter() {
 }
 
 // InstallTracingFilter installs OpenTelemetry tracing filter with the restful Container.
-func (s *Server) InstallTracingFilter(tp oteltrace.TracerProvider) {
-	s.restfulCont.Filter(otelrestful.OTelFilter("kubelet", otelrestful.WithTracerProvider(tp)))
+func (s *Server) InstallTracingFilter(tp oteltrace.TracerProvider, opts ...otelrestful.Option) {
```

Comment:
Does it make sense to keep the method like this? Since traceProvider also only becomes an option. (`otelrestful.WithTracerProvider(tp`)

```suggestion
func (s *Server) InstallTracingFilter(opts ...otelrestful.Option) {
```

Reply:
I would actually prefer to keep tp as an explicit argument to ensure it is passed.  We want to make sure we avoid falling back to the global tracer provider.

Reply:
Ok, I will quickly revert it.